### ISSUE TITLE
Segmentation fault when installing mongodb

### ISSUE THREAD

**Issue by greg0ire** [shivammathur/setup-php#348]
**Describe the bug**
The output is cluttered with warnings, and there is a segmentation fault:

```
==> Setup Extensions
/home/runner/work/_actions/shivammathur/setup-php/v2/dist/../src/scripts/common.sh: line 138: 13465 Broken pipe             yes '' 2> /dev/null
     13466 Segmentation fault      | sudo pecl install -f "$extension" > /dev/null 2>&1
PHP Warning:  Module 'mongodb' already loaded in Unknown on line 0
✓ mongodb Installed and enabled
PHP Warning:  Module 'mongodb' already loaded in Unknown on line 0
PHP Warning:  Module 'mongodb' already loaded in Unknown on line 0
PHP Warning:  Module 'mongodb' already loaded in Unknown on line 0
```

**Version**
- [x] I have checked releases, and the bug exists in the latest patch version of `v1` or `v2`.
- [x] `v2`
- [ ] `v1`

**Runners**

- [x] GitHub Hosted
- [ ] Self Hosted

**Operating systems**
ubuntu-20.04

**PHP versions**
PHP 7.2, 7.3 and 7.4 have the issue

**To Reproduce**
```
  phpunit-standalone:
    name: "PHPUnit on standalone servers"
    runs-on: "${{ matrix.os }}"

    strategy:
      matrix:
        os:
          - "ubuntu-20.04"
        php-version:
          - "7.2"
          - "7.3"
          - "7.4"
        mongodb-version:
          - "4.4"
          - "4.2"
          - "4.0"
          - "3.6"
        driver-version:
          - "stable"
        deps:
          - "normal"
        include:
          - deps: "low"
            os: "ubuntu-20.04"
            php-version: "7.2"
            mongodb-version: "3.6"
            driver-version: "1.5.0"
      - name: "Install PHP"
        uses: "shivammathur/setup-php@v2"
        with:
          php-version: "${{ matrix.php-version }}"
          tools: "pecl"
          extensions: "mongodb-${{ matrix.driver-version }}, bcmath"
          coverage: "none"
          ini-values: "zend.assertions=1"
```

**Expected behavior**
No segmentation fault

**Screenshots/Logs**
https://github.com/doctrine/mongodb-odm/runs/1462657688?check_suite_focus=true#step:6:21

**Are you willing to submit a PR?**
No

**Comment by greg0ire** [comment]
Related: https://github.com/mongodb/mongo-php-driver/issues/1177 Feel free to close this since I suppose.

**Comment by alcaeus** [comment]
See also the upstream bug report: https://bugs.php.net/bug.php?id=80418.

One workaround would be to run `pecl uninstall <extension>` if it's already installed, or removing the `extension=<extension>.so` line from php.ini. This would also remove the warnings due to pecl adding a second `extension=` line and PHP attempting to load the extension multiple times.

**Comment by shivammathur** [comment]
@greg0ire This is a known bug, when new version of extension is loaded and there is a previous version, there is a segmentation fault. 

There is a workaround for this in `setup-php`, just that 'extension-stable' does not trigger that workaround. Working on a patch to fix this.

**Comment by shivammathur** [comment]
@greg0ire @alcaeus Fixed in 1d47b6b

**Comment by alcaeus** [comment]
Thanks for the quick fix, it is very much appreciated!
